NitroxClient/GameLogic/InitialSync/InventoryItemsInitialSyncProcessor.cs

```diff
@@ -51,7 +51,7 @@ public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualW
                     }
                     catch (Exception ex)
                     {
-                        Log.Error("Error deserializing item data " + itemdata.ItemId + " " + ex.Message);
+                        Log.Exception("Error deserializing item data " + itemdata.ItemId, ex);
```

Comment:
I think it's best to have only `Log.Error` and have an overload that accepts exception object. This makes it consistent with the other methods that are also named after the log level they output on.